test/lstm_common.hpp

```diff
@@ -2928,6 +2928,9 @@ struct lstm_basic_driver : test_driver
     {
 
 #if(MIOPEN_BACKEND_OPENCL == 1)
+        std::cout << "Skip test for Issue #692: " << std::endl;
```

Comment:
I think we shall 
- formalize workaround
- use exit(...)
- avoid "unreachable code" warnings

Reply:
1. What is current formal workaround format?
2.  will do
3.  where does the "unreachable code" warnings happen?

Reply:
Thanks!